packages/openapi-fetch/src/index.ts

```diff
@@ -51,27 +51,21 @@ export interface DefaultParamsOption {
   params?: { query?: Record<string, unknown> };
 }
 
-export interface EmptyParameters {
-  query?: never;
-  header?: never;
-  path?: never;
-  cookie?: never;
-}
-
 export type ParamsOption<T> = T extends { parameters: any }
   ? HasRequiredKeys<T["parameters"]> extends never
     ? { params?: T["parameters"] }
     : { params: T["parameters"] }
-  : never;
+  : DefaultParamsOption;
+// v7 breaking change: TODO uncomment for openapi-typescript@7 support
+// : never;
 
 export type RequestBodyOption<T> = OperationRequestBodyContent<T> extends never
   ? { body?: never }
   : undefined extends OperationRequestBodyContent<T>
   ? { body?: OperationRequestBodyContent<T> }
   : { body: OperationRequestBodyContent<T> };
 
-export type FetchOptions<T> = RequestOptions<T> &
-  Omit<RequestInit, "body"> & { fetch?: ClientOptions["fetch"] };
```

Comment:
Doesn’t change the option itself; just moved to be with the other options (it was inconsistent here)